core/base/libs/socrate/socrate/system.py

```diff
@@ -38,6 +41,17 @@ def set_env(required_secrets=[]):
     for secret in required_secrets:
         os.environ[f'{secret}_KEY'] = hmac.new(bytearray(secret_key, 'utf-8'), bytearray(secret, 'utf-8'), 'sha256').hexdigest()
 
+    try:
+        ip_resolver = dns.resolver.query("resolver")[0].address
+        log.info(f'Setting DNS to {ip_resolver}')
+        with open('/etc/resolv.conf','w') as f:
+            f.write(f'nameserver {ip_resolver}\noptions ndots:0\n')
+    except:
+        pass
```

Comment:
Can this be enhanced with a log message as well?
Maybe: 
`
log.info('The unbound container with name resolver cannot be found or is not enabled. Falling back to docker embedded DNS server')
`

Reply:
Done in 1da7b28d